Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
> @chrmoritz Everything is fixable. The question is what else this breaks that I haven't randomly stumbled over yet.

I still think you did something in a gray area which was unsupported by npm and even grafana upstream (their doc advices you to install grunt-cli globally with npm, which does not cause the issue with running npm on a yarn created dep tree and yes I know this is not something which should be done inside homebrew). You simple can't demand from npm that it is able to add something to an unknown arbitary strutured dep tree, without breaking the stuff inside the unknown tree, while still resolving all deps (the requested plus the unknown ones) correctly and doing deduping and stuff...

> @chrmoritz did you figure out why the initial yarn install fails now since that actually happens before the npm install?

Nope, I don't have access to my mac until later today and it works fine for my outside homebrew. Looks like yarn can't find node-gyp (which still comes bundled with npm). Maybe it's related to npm installation change in this formula. Does the bignum test from inside `brew test node` pass for you or does it fail too?